test/libjulia/Inliner.cpp

```diff
@@ -342,5 +342,40 @@ BOOST_AUTO_TEST_CASE(pop_result)
 	);
 }
 
+BOOST_AUTO_TEST_CASE(inside_condition)
+{
+	// This tests that breaking the expresison inside the condition works properly.
```

Comment:
Typo: expression

Reply:
I think I'm making this exact typo way too often...